### ISSUE TITLE
[Feature request] Minimal search

### ISSUE THREAD

**Issue by alexsasharegan** [pop-os/pop#573]
**Distribution (run `cat /etc/os-release`):**
 
```txt
NAME="Pop!_OS"
VERSION="19.04"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 19.04"
VERSION_ID="19.04"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=disco
UBUNTU_CODENAME=disco
``` 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

System Search (from keyboard: `SUPER`)

**Feature Description: _Quick Search_**

I would love to see a minimal search overlay that does not include the additional functionality of the current behavior when pressing **super**--essentially a quick search. I don't want to change existing behavior, but add this as a new feature.

The model for this type of quick search is like alfred for mac ([alfredapp.com](https://www.alfredapp.com/)) or the vscode command palette (via **F1**, **ctrl/cmd+p**). Instead of taking control of the whole screen, it would just introduce a small overlay window above all existing windows. The width of the window would be about the width of the current Notification Center window, and placement would be horizontally and vertically centered. It would use the same backend for search that **super** uses, but would display results in a more compact format. Ideal keybinding would be **super + space**.

The need for this feature is that currently pressing **super** is a very big visual shift from your current task. When there are many windows open and multiple workspaces, the screen animates a great deal, and on larger monitors, the search input is the least conspicuous element in view. A dedicated quick search would not affect any open windows except that it would sit on top of them and command attention visually. It could also be much faster/more responsive. The referenced implementations of quick search do not animate their opening, so they have an extremely responsive feel that encourages usage of the tool. It becomes faster to search for an open app than to use **alt + tab** and tab through multiple entries to select it.

**Notes**

One sidecar feature request: add math capabilities to the search input. Submit via enter copies to clipboard. This is another alfred feature that ends up being surprisingly useful, but isn't really part of this request.

**Current Search Screen**

![windowzzz](https://user-images.githubusercontent.com/16269252/58339100-b6e26900-7dfd-11e9-858f-df7d28032f18.png)

**Alfred Search**

![image](https://user-images.githubusercontent.com/16269252/58347783-9cb38580-7e13-11e9-849a-dc7d6fb0dd6c.png)

**VSCode Command Palette**

![image](https://user-images.githubusercontent.com/16269252/58347921-f451f100-7e13-11e9-957f-e8c6e7753976.png)

**Comment by point-source** [comment]
Sadly, I don't think we know any of the same people :/ 

I wanted to drop in and make sure you intended to post screenshots with potentially personal info. Just a friendly heads up in case ;)

**Comment by alexsasharegan** [comment]
Appreciate the heads up. Maybe you could be more specific though? The first screenshot with signal? I gave a look through things, and I don't think I'm concerned with anything exposed, but maybe I missed something.

**Comment by point-source** [comment]
Nope, I didn't see any thing especially egregious. Some people are more paranoid than others. I think you're fine. :ok_hand:

**Comment by alexsasharegan** [comment]
I figured you're probably right on a couple spots, so I updated that screecap with a redacted version 😄

**Comment by leviport** [comment]
This sounds a lot like our launcher. I think this feature request can be closed now.
